util/build_configs/README.md

```diff
@@ -27,6 +27,46 @@ Work-in-progress (2018-07-31)
   logs, CHPL_HOME workspaces, etc in this directory, without adding
   them to Git.
 
+  Let's say your host machine has Slurm; you want a Chapel multi-config
+  build with and without Gasnet (where Gasnet may use either of two widely-
+  supported comm layers); and you also want the option to use the native
+  Slurm launcher slurm-srun as well as Chapel's default Gasnet launcher(s).
+  Let's call your new project "slurm".
+
+  The existing setenv-example-1.bash script seems quite similar, except for
+  using Slurm.
+
+  To get started, copy setenv-example-1.bash to a new file "slurm.bash" and
+  add an additional command line parameter "--launcher=UNSET,slurm-srun" to
+  the first build_configs.py command in the file.
+
+  This will add another Chapel make (with CHPL_LAUNCHER=slurm-srun) to each
+  existing Gasnet config. It will also add another Chapel make (with slurm-srun)
+  to the existing non-Gasnet (CHPL_COMM=none) config- a small waste of time!
+  You might want to add something to the setenv callback to skip useless builds
+  with CHPL_COMM=none and CHPL_LAUNCHER=<any non-null value>. But, its optional.
+
+  Finally, replace all references to "setenv-example-1.bash" in your file with
```

Comment:
Could we save `setenv-example-1.bash` to a variable, so that the user only needs to change a single line for this modification? Going further, we could grab the name of the script with `basename "$0"`, so that this change is not necessary.

Reply:
Yes, that is the style shown in setenv-example-2. In fact, setenv-example-1 was originally written that way. Later, I decided setenv-example-1 should be much simpler, to not distract the reader from the important build_configs / setenv callbacks pattern.  So, I stripped-out as much of that bashy goodness as I could!  I didn't anticipate trying to describe how to copy-and-change any of these examples in this level of detail. 

We could avoid telling the user to edit-change "setenv-example-1" references to "slurm", just by using setenv-example-2 instead. Because that one already does what you suggest.  Let me take a look at that.